utils/gen.js

```diff
@@ -7,9 +7,14 @@
 const { writeFileSync } = require('fs');
 const { join } = require('path');
 
-const { versions, platform, arch } = process;
+const { versions, platform, arch, env } = process;
 const file = join(__dirname, '..', 'turbo-cache-key.json');
 const node = versions.node.split('.')[0];
-const str = JSON.stringify({ node, platform, arch });
+const str = JSON.stringify({
+  node,
+  platform,
+  arch,
+  imageVersion: env.ImageVersion ?? 'local',
```

Comment:
Did we verify that this exists on Windows?

Reply:
I did not. I was hoping to see it in the CI during this PR

Reply:
Oh good call. Nice, looks like it's present on all:

```
Generating cache key: {"node":"16","platform":"darwin","arch":"x64","imageVersion":"20231115.2"}
```

```
Generating cache key: {"node":"16","platform":"win32","arch":"x64","imageVersion":"20231115.2.0"}
```

```
Generating cache key: {"node":"16","platform":"linux","arch":"x64","imageVersion":"20231115.7.0"}
```